Packs/Slack/Integrations/SlackV3/README.md

```diff
@@ -571,3 +571,18 @@ Because of this, you may have some users in Slack who are not users in XSOAR. Th
 health to indicate that an error has occurred because a user was unable to be found. In this circumstance, the error is expected and is purely cosmetic in nature.
 - In some cases when mirroring an investigation, kicking the admin will cause no further actions to be able to be performed by the bot. Any subsequent actions taken on the channel (such as mirror out) will result in a "not in channel" error.
 - Please note: If a dedicated channel is configured, however there are no notifications being sent, please verify that the **Types of Notifications** to send parameter is populated.
+### slack-get-integration-context
```

Comment:
I think this is out of order. We can just leave it out since it's an internal debug command however.

Reply:
I was getting issues from the validator that it wasnt in the readme. Is it an issue to leave? wdym out of order? Is there a better spot for it?